Better docs for gitlab token (#131)
env0/terraform-provider-env0: docs/resources/template.md

- **revision** (String) source code revision (branch / tag) to use
- **ssh_keys** (List of Map of String) an array of references to 'data_ssh_key' to use when accessing git over ssh
- **terraform_version** (String) Terraform version to use

> - **token_id** (String) The token id used for private git repos or for integration with GitLab, you can get it by using data resource on existing template or contact us

Reviewer: ```suggestion
- **token_id** (String) The token id used for private git repos or for integration with GitLab, you can get it by using data resource of an existing template or contact our support team
```

> - **token_id** (String) The token id used for private git repos or for integration with GitLab, you can get this value by using data resource of an existing Gitlab template or contact our support team

Reviewer: ```suggestion
- **token_id** (String) The token id used for private git repos or for integration with GitLab, you can get this value by using a data resource of an existing Gitlab template or contact our support team
```
